scripts/test-helpers

```diff
@@ -234,7 +234,6 @@ sonobuoy-test() {
     sonobuoy run \
         --config=scripts/sonobuoy-config.json \
         --plugin-env=e2e.E2E_USE_GO_RUNNER=true \
-        --sonobuoy-image=rancher/sonobuoy-sonobuoy:v${SONOBUOY_VERSION:-0.19.0} \
```

Comment:
What's the logic for removing this line instead of merely updated the default value? It seems useful to be able to override. Or is this still possible but achieved via other mechanisms @brandond ?

Reply:
The CLI tool always expects the image version to match the tool version; we were only doing this to get around the docker hub rate-limiting but that is not necessary now that we have a pull-through registry cache. Dropping this removes one more step (getting a pr merged to rancher/image-mirror) when we need to update sonobuoy.